src/pretix/settings_helpers.py

```diff
@@ -0,0 +1,38 @@
+def build_db_tls_config(config, db_backend):
+    db_ssl_mode = config.get("database", "sslmode", fallback="disable")
+    # add postgresql TLS options
+    if db_ssl_mode != "disable" and db_backend == "postgresql":
+        db_tls_config = {
+            "sslmode": db_ssl_mode,
+            "sslrootcert": config.get("database", "sslrootcert"),
+        }
+        # add postgresql mTLS options
+        if config.has_option("database", "sslcert"):
+            db_tls_config.update(
```

Comment:
I thinks in these lines, writing like this is clearer:

```py
db_tls_config['sslcert'] = config.get('database', 'sslcert')
```

Reply:
I have made the changes.